__tests__/author.test.js

```diff
@@ -0,0 +1,65 @@
+const pool = require('../lib/utils/pool');
+const setup = require('../data/setup');
+// const { request } = require('express');
+const request = require('supertest');
+const app = require('../lib/app');
+
+describe('authors route', () => {
+  beforeEach(() => {
+    return setup(pool);
+  });
+  it('/authors should return a list of authors', async () => {
+    const res = await request(app).get('/authors');
+    // eslint-disable-next-line
+    const expected = [{
+      id: 1,
+      name: 'Aldous Huxley',
+      dob: 1894,
+      pob: 'Surrey, England',
+      books: expect.any(Array)
+    },
+    {
+      id: 2,
+      name: 'Eric Arthur Blair',
+      dob: 1903,
+      pob: 'Motihari, British India',
+      books: expect.any(Array)
+    }
+    ];
+    
+    expect(res.body[2]).toEqual({
```

Comment:
because you're using this object inside your toEqual - you can delete your expected variable in the scope of this test, since it's no longer being used